google-cloud-spanner/lib/google/cloud/spanner/session.rb

```diff
@@ -54,11 +54,16 @@ class Session
         # @private The hash of query options.
         attr_accessor :query_options
 
+        ##
+        # @private The hash of directed read options.
+        attr_accessor :directed_read_options
+
         # @private Creates a new Session instance.
-        def initialize grpc, service, query_options: nil
+        def initialize grpc, service, query_options: nil, directed_read_options: nil
           @grpc = grpc
           @service = service
           @query_options = query_options
+          @directed_read_options = directed_read_options
```

Comment:
Is this the client level directed read option?

Reply:
same as comment above